packages/shared/lib/auxiliary/deep-link/handlers/wallet/operations/getAmountFromSearchParam.ts

```diff
@@ -0,0 +1,16 @@
+import { AmountNotANumberError, SendOperationParameter } from '@auxiliary/deep-link'
+import { ITokenMetadata, parseRawAmount } from '@core/wallet'
+
+export function getAmountFromSearchParam(searchParams: URLSearchParams, tokenMetadata: ITokenMetadata): string {
```

Comment:
Because this isn't a specific operation handler function, we could probably create a `utils` folder within the `deep-link` module for this? Or a helper.

Reply:
What's the difference between a `helper` or `utils` module?